docs/guide/comparisons.md

```diff
@@ -42,7 +42,7 @@ WebdriverIO comes with the same advantages as Cypress allowing you to test your
 
 [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/) runs tests inside a headless browser, providing the same execution environment as your web application without the need for mocking out browser APIs or the DOM. This also makes it possible to debug inside a real browser using the devtools, although there is no UI shown for stepping through the test, as there is in Cypress tests.
 
-There is a watch mode, but it is not as intelligent as that of Vitest, and may not always re-run the tests you want.
+There is a watch mode, but it is not as intelligent as that of Vitest, and may not always re-run the tests you want.<sup>[[citation needed](https://en.wikipedia.org/wiki/Wikipedia:Citation_needed)]</sup>
```

Comment:
This was added in 2021 - maybe this is simply outdated? https://github.com/vitest-dev/vitest/pull/128

I'm afraid if we start adding these `citation needed` labels, the whole docs would eventually fill up with them.